action/remove.go

```diff
@@ -2,24 +2,70 @@ package action
 
 import (
 	"os"
+	"os/exec"
 	"path/filepath"
+	"strings"
 
 	"github.com/helm/helm/log"
+	"github.com/helm/helm/manifest"
 )
 
+// kubeGet executes the kubectl command, override in tests
+var kubeGet func(string) string
```

Comment:
The most idiomatic way to do this would be to add a type:

```go
type kubeGetter func(string) string
var kubeGet kubeGetter
```

A good example in `"net/http".HandlerFunc`.